assets/src/styles/blocks/Columns.scss

```diff
@@ -4,32 +4,28 @@
       font-weight: inherit;
     }
 
-    &.btn-primary {
-      color: $white;
-    }
-
     &.btn-secondary {
       width: 100%;
     }
   }
 
   h3, h3 > a {
     line-height: 1.6rem;
-    font-size: $font-size-lg;
-    font-family: $roboto;
-    font-weight: 700;
 
     --block-columns--column-heading-- {
+      font-size: $font-size-lg;
+      font-family: $roboto;
+      font-weight: 700;
       color: $grey-80;
-    }
+      margin-bottom: .5rem;
```

Comment:
Campaigns customize this bottom margin, but we didn't have a property for it. For now I added it with the value it effectively has.